doc/classes/Label.xml

```diff
@@ -62,8 +62,8 @@
 		</member>
 		<member name="mouse_filter" type="int" setter="set_mouse_filter" getter="get_mouse_filter" overrides="Control" enum="Control.MouseFilter" default="2" />
 		<member name="percent_visible" type="float" setter="set_percent_visible" getter="get_percent_visible" default="1.0">
-			Limits the number of visible characters. If you set [code]percent_visible[/code] to 0.5, only up to half of the text's characters will display on screen. Useful to animate the text in a dialog box.
-			[b]Note:[/b] Setting this property updates [member visible_characters] based on current [method get_total_character_count].
+			The fraction of characters to display, related to the total character count (see [method get_total_character_count]). If [code]1.0[/code], all characters display. If [code]0.5[/code], only up to half the characters will display. Useful to animate the text in a dialog box.
```

Comment:
```suggestion
			The fraction of characters to display, relative to the total number of character (see [method get_total_character_count]). If set to [code]1.0[/code], all characters are displayed. If set to [code]0.5[/code], only half of the characters will be displayed. This can be useful when animating the text appearing in a dialog box.
```

And the same for RTL, I assume.